src/app/pages/IdxPage/index.test.jsx

```diff
@@ -34,6 +35,37 @@ describe('IdxPage', () => {
     });
   });
 
+  describe('Assertions', () => {
+    it('should render idx page sections', async () => {
+      const sections = container.querySelectorAll('section');
+      expect(sections.length).toBeGreaterThan(1);
+      sections.forEach(section => {
+        expect(section.getAttribute('role')).toEqual('region');
+      });
+    });
+  });
+});
+
+describe('IdxPage - Ukrainian', () => {
+  beforeEach(async () => {
+    fetchMock.restore();
+
+    await act(async () => {
+      container = render(
+        <IdxPageWithContext
+          service="ukrainian"
+          pageData={ukraineInRussianIdxData}
+        />,
+      ).container;
+    });
+  });
+
+  describe('Snapshots', () => {
+    it('should render a ukrainian idx page correctly with most read and radio schedule', async () => {
```

Comment:
Ukrainian IDX doesn't have Most Read and Radio Schedule 🤔

Reply:
I've removed this text now `with most read and radio schedule`